models/fixtures/user.yml

```diff
@@ -218,3 +218,33 @@
   avatar_email: user13@example.com
   num_repos: 3
   is_active: true
+
```

Comment:
Is there a reason we can't just use existing user fixtures? user16 isn't used anywhere, and it seems we could use any existing user in place of user15 in `TestSignin`.